### ISSUE TITLE
🗼AUR Support

### ISSUE THREAD

**Issue by DuckSoft** [database64128/shadowsocks-uri-generator#2]
Gonna make this into AUR. Which name do you prefer?
- `ss-uri-gen`
- `shadowsocks-uri-generator`
- other
@database64128

**Comment by database64128** [comment]
Thank you for the offer. I'd love to see this project packaged into AUR.

Let's name the package `shadowsocks-uri-generator`, so anyone searching for `shadowsocks` can easily find it.

`ss-uri-gen` is the binary name, and the directory name in `/opt`, so it's easy to remember and type.

How do you plan to package it? Do you plan to use framework-dependent packaging and depend on `dotnet-sdk-bin`, or self-contained packaging that doesn't depend on any package?

Where should we place the configuration files? Currently it's hardcoded to use the current working directory.

You might be interested in https://aur.archlinux.org/packages/powershell/.

**Comment by database64128** [comment]
Changes made in 4bebeae.

cc @kidonng

**Comment by database64128** [comment]
@kidonng I just noticed https://github.com/kidonng/sushi/commit/b5ad57462142090f56cfa86af60a8a6fa242598f. Thank you for the effort. But you kind of missed the point. 🤣

What I intended to let you notice is https://github.com/database64128/shadowsocks-uri-generator/commit/4bebeae32eade11bccf80d3f1a8fd0afa770ba56, in which the behavior of choosing the config path is altered. Without the compilation constant `PACKAGED` defined, the default behavior is to use executable directory as config folder. By defining `PACKAGED`, the behavior is changed to use user config directory.

It certainly seems like there is no way for your bucket to initiate a custom build. Should I include a custom build in the release, say `ss-uri-gen-v3.3-windows-scoop.7z`?
